jcommune-view/jcommune-web-view/src/main/webapp/resources/css/app/application.css

```diff
@@ -486,7 +486,12 @@ div.anchor {
     margin-left: 20px;
 }
 
-#signup-modal-dialog .captcha-images img {
+/*#signup-modal-dialog .captcha-images img {
+    vertical-align: middle;
+}*/
+
+/*align captcha image with buttons on line, registration page*/
```

Comment:
This style apply to registration form in modal and stand alone window.
I'm not exactly understand your last comment. Style related to registration page/modal, is it wrong? Should I change comment? ))